packages/xo-server-backup-reports/src/index.js

```diff
@@ -317,6 +317,7 @@ class BackupReportsXoPlugin {
         const taskMarkdown = await getMarkdown(task, {
           formatDate,
           jobName: log.jobName,
+          xo,
```

Comment:
Was it broken before?

Reply:
No, we never had a task of type remote in this level.